app/components/utils/government-areas/government-areas-panel.hbs

```diff
@@ -24,8 +24,21 @@
       </Auk::Toolbar::Group>
     </Auk::Toolbar>
   </Auk::Panel::Header>
-  {{#if @governmentAreas.length}}
-    <Auk::Panel::Body @noPadding={{true}}>
+  <Auk::Panel::Body @noPadding={{if @governmentAreas.length true false}}>
+    {{#if @subcase.parliamentRetrievalActivity.themes}}
+      <div class="auk-u-m-4 auk-u-mb-2 auk-u-mt-2">
+        <p><strong>{{t "themes"}}: </strong>
+          <FormatList
+            @list={{@subcase.parliamentRetrievalActivity.themes}}
+            @locale="nl-be"
+            as |formattedPart|
+          >
+            {{formattedPart.value}}
+          </FormatList>
+        </p>
+      </div>
+    {{/if}}
+    {{#if @governmentAreas.length}}
```

Comment:
doing this in the template will now try `subcase.parliamentRetrievalActivity` on every subcase where this util is used. but you only really need it for 1 subcase type.
It would make more sense to get these themes in the parent component and passing those down instead.
it avoid extra calls/load to the frontend